js/packages/openinference-core/src/types.ts

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type GenericFunction = (...args: any[]) => any;
```

Comment:
we can remove these and the withSafety util below from other packages if we want to export all of this stuff out of this package

Reply:
let's move this into the /utils directory